### DISCUSSION TITLE
Introduce publish step to deploy built packages to a registry

### DISCUSSION THREAD

**Discussion by alex-kovalenko1982** [quarkiverse/quarkus-quinoa#756] [Ideas]
Would be great to introduce additional step in the quinoa package manager commands section to enable publishing built UI module to a local/remote registry. This is especially useful for modules that contain only some shared angular/react components that shall be only built and published to be further included as dependencies into other modules but not run or served standalone. 

Suggestion for new config property:
`quarkus.quinoa.package-manager-command.publish` that gets value to construct deploy command like `npm publish --arguments`

if possible the new command shall be linked to deploy phase of the maven/gradle tool.

**Comment by melloware** [comment]
@ia3andy thoughts?

**Reply by ia3andy** [reply]
I don't think we can hook into the deploy phase of the build tools?

**Reply by ia3andy** [reply]
I would be ok adding a new config named quinoa.publish or something that when enabled, disable quinoa at runtime and call a new publish command

**Reply by alex-kovalenko1982** [reply]
could you please check the PR https://github.com/quarkiverse/quarkus-quinoa/pull/781/files as a follow-up of this discussion.
